tex-exchange-contract/src/main/kotlin/com/ccc/state/Order.kt

```diff
@@ -0,0 +1,36 @@
+package com.ccc.state
+
+import com.ccc.contract.OrderContract
+import net.corda.core.contracts.Amount
+import net.corda.core.contracts.BelongsToContract
+import net.corda.core.contracts.LinearState
+import net.corda.core.contracts.UniqueIdentifier
+import net.corda.core.identity.Party
+import net.corda.core.serialization.CordaSerializable
+import java.time.Instant
+import java.util.*
+
+@CordaSerializable
+enum class Direction { SELL }
```

Comment:
Should we have `BUY` as well?

Reply:
Buy Orders are not in scope at the moment